provisioning/provisioning-service-client/pom.xml

```diff
@@ -38,25 +38,12 @@
             <artifactId>iot-deps</artifactId>
             <version>0.7.1</version>
         </dependency>
-        <!-- test dependencies -->
-        <dependency>
-            <groupId>org.jmockit</groupId>
-            <artifactId>jmockit</artifactId>
-            <version>1.22</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit-dep</artifactId>
-            <version>4.11</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.hamcrest</groupId>
-            <artifactId>hamcrest-all</artifactId>
-            <version>1.3</version>
-            <scope>test</scope>
-        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.microsoft.rest/client-runtime -->
+		<dependency>
+		    <groupId>com.microsoft.rest</groupId>
+		    <artifactId>client-runtime</artifactId>
+		    <version>1.6.2</version>
+		</dependency>
```

Comment:
The provisioning service client requires ms autorest; it is also required in deps since it is required by ServiceClientCredentials class (which is used for authentication)

Reply:
Actually I take it back, you're right. Including ms autorest in deps, and having deps as dependency of provisioning service client works!